src/Core/Utility.js

```diff
@@ -203,3 +206,32 @@ function hash (str) {
 
 	return code >>> 0
 }
+
+/**
+ * @param {function} callback
+ * @return {number}
+ */
+function timeout (callback) {
+	return setTimeout(function () {
+		callback(now())
```

Comment:
As much as I like correctness in polyfills, `Date.now()` is an expensive operation for a value that is never used